telco-examples/edge-clusters/airgap/README.md

```diff
@@ -0,0 +1,201 @@
+* In the following examples, we are assuming that the edge cluster will use Baremetal Servers. If you want to deploy the full workflow using virtual machines, please refer to the [metal3-demo repo](https://github.com/suse-edge/metal3-demo)
+* In the following examples, as we're creating an air-gap scenario, we are assuming that you have a local private registry deployed in the same network. 
+
+## Create the image for the edge cluster
+
+### Prerequisites
+
+To prepare an airgap environment, we will use a private local registry (already deployed and configured) to store some images (helm-chart oci images and some other images), and we will use the Edge Image Builder to include the rke2-images artifacts which are a requirement to use the cluster-api provider for rke2.
+
+Using the example folder `telco-examples/edge-clusters/airgap/eib` we will create the basic structure in order to build the image for the edge cluster: 
+
+You need to modify the following values in the `telco-edge-airgap-cluster.yaml` file:
+
+- `${ROOT_PASSWORD}` - The root password for the management cluster. This could be generated using `openssl passwd -6 PASSWORD` and replacing PASSWORD with the desired password, and then replacing the value in the `telco-edge-cluster.yaml` file.
+- `${SCC_REGISTRATION_CODE}` - The registration code for the SUSE Customer Center for the SLE Micro product. This could be obtained from the SUSE Customer Center and replacing the value in the `telco-edge-cluster.yaml` file.
+
+You need to modify the following folder:
+
+- `base-images` - To include inside the `SLE-Micro.x86_64-5.5.0-Default-RT-GM.raw` image downloaded from the SUSE Customer Center.
+
+### preparing the airgap aritfacts
+
+The following steps are required to prepare the airgap artifacts:
+
+1.Include the rke2-release-images to the `custom/files` folder to be consumed by EIB during the build process. 
+  - You can use the [following script](https://github.com/suse-edge/fleet-examples/blob/release-3.0/scripts/day2/edge-save-rke2-images.sh) and the list of images [here](https://github.com/suse-edge/fleet-examples/blob/release-3.0/scripts/day2/edge-release-rke2-images.txt) to generate the artifacts required to be included in `custom/files`. 
+  ```
+  $ ./edge-save-rke2-images.sh -o ~/telco-examples/edge-clusters/airgap/eib/custom/files -l ~/edge-release-rke2-images.txt
+  ...
+  $ tree ~/telco-examples/edge-clusters/airgap/eib/custom/files
+  .
+  |-- install.sh
+  |-- rke2-images-cilium.linux-amd64.tar.zst
+  |-- rke2-images-core.linux-amd64.tar.zst
+  |-- rke2-images-multus.linux-amd64.tar.zst
+  |-- rke2-images.linux-amd64.tar.zst
+  |-- rke2.linux-amd64.tar.gz
+  `-- sha256sum-amd64.txt
+  ```
+
+2. Preload your registry with the helm-chart oci images required for the edge cluster. 
```

Comment:
```suggestion
2. Prepare your artifact registry.
```

Reply:
mmm I'd like to keep the preload. it's really common when you're working with disconnected environments